### ISSUE TITLE
Is this project maintained or can we use this for production usage?

### ISSUE THREAD

**Issue by magiccDev** [Askedio/laravel-ratchet#37]
Is this project maintained or can we use this for production usage?

Looking forward to hearing from you

**Comment by gcphost** [comment]
This project is not actively maintained, however, I still accept PR's that are valid, our last one was for Laravel 6.0.

I am looking for new maintainers. The package has great potential and works well with Laravel 5/6.

**Comment by magiccDev** [comment]
Great and thank for the reply @gcphost . Shall we create a roadmap. I would like to contribute.

**Comment by gcphost** [comment]
Certainly, feel free to use this thread, start a new one, or send me a DM :D

**Comment by magiccDev** [comment]
How do I contact? do you have Slack?

**Comment by gcphost** [comment]
I do: https://join.slack.com/t/playerkings/shared_invite/zt-hw920hgg-Bmib~DpIUhQKSEW__54gBA you can DM me there, Will.

**Comment by ghost** [comment]
This will not install on Laravel 6.20.2 

```
Problem 1
    - askedio/laravel-ratchet[5.6, ..., 5.6.x-dev] require illuminate/console ^5.5 -> found illuminate/console[v5.5.0, ..., 5.8.x-dev] but it conflicts with another require.
    - Root composer.json requires askedio/laravel-ratchet ^5.6 -> satisfiable by askedio/laravel-ratchet[5.6, 5.6.1, 5.6.2, 5.6.x-dev].
```

I'd really like to be able to get something running to see if ratchet will even do what I want ... should I downgrade to Laravel 5.5?

**Comment by gcphost** [comment]
Looks like the package has laravel 6 support added in some regards, its possible the package was not updated properly. I'm happy to accept any PRs as I have no way to check this project at this time.

**Comment by ghost** [comment]
Thanks,

I appreciate the quick response.
